tests/common/snappi_tests/traffic_generation.py

```diff
@@ -578,7 +578,8 @@ def verify_in_flight_buffer_pkts(duthost,
                               format(dropped_packets))
 
 
-def verify_pause_frame_count_dut(duthost,
+def verify_pause_frame_count_dut(rx_dut,
```

Comment:
This function is also used in tests/snappi_tests/pfc/files/helper.py. Maybe change that part too.

Reply:
Yes. I have made changes to helper.py as well.